moto/swf/models/domain.py

```diff
@@ -31,6 +32,10 @@ def to_short_dict(self):
         hsh = {"name": self.name, "status": self.status}
         if self.description:
             hsh["description"] = self.description
+        region = self.swf_backend.region_name if self.swf_backend else "unknown"
```

Comment:
We should always know which region we're in, so the backend/region should never be None

Reply:
Done. It was originally optional because I didn't see an obvious region to use for testing... but I just picked a test region like other testing modules.